pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "wfdb"
+version = "4.0.0-alpha0"
```

Comment:
Are you intentionally bumping the version here?

Reply:
Yes, I want to make sure this works if I upload it to pypi. Hence, the alpha version.

The way this package is installed is going to be different (users will explicitly have to specify processing/plot to install those deps), so it will be a new major version (coupled with other changes).